notebooks/minigrid_colab_tutorial.ipynb

```diff
@@ -0,0 +1,560 @@
+    "The reward provided to the agent on reaching the green square is penalized by number of steps it took to reach there.\n",
+    "\n",
+    "<!-- About the environment - We are using `MiniGrid-Empty-5x5-v0` environment which is an empty room and the objective of the agent is to reach the green sqare, the reward provided to the agent on reaching the green square is penalized and dependent on how many steps it took to reach there. -->\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOIAAADfCAMAAADcKv+WAAAAwFBMVEVMTExkZGSSkpL///8AAAAA/wBHR0dbW1tJSUlnZ2eVlZWIiIhNTU1ERETb29vw8PB7e3vPz89ycnL/TEwgICA9PT01NTUMDAxXV1dpU2kK/AonJycAXQAATABESUl8TEyKioqgoKDm5ubDw8OsrKy2trawsLDl5eW/v794eHhWTEw6TEz3TEzYTEwZGRlnTExxTEyjTEzqTEyXTEyJTEzcTEy8TEzSTEzCTEyuTEwxPDFOiE5pZGlYU1hOek4KAApxiaKtAAAFcElEQVR4nO3dC1ebSBgG4AljE5hJGKBe0u0WGS4hRrtqb3trd///v1qidSPKJ5McFJm87+mx1o/LPBkglcwZGLc+rO8GPH/2i+iLsr92PGPuESVPqq++XakTgwoZBsfu2J64h6pGzPMF5+HUG9kTb1Yn8uqfPGR9N6vTPCRyEAcYEG0IiDYERBsCog0B0YaAaENAtCEg2hAQbci+Et2+W9VpGoleQEVuH3oduuKRGdGVrYiBcJojzqgdBOSuk4iqCFpyTORoQlWOSWMzkRA6YjR2XTeup/qJO45cIjM5oUpiRlW8AyKHE6py0BXRcxlbndZzyRhzoylrzlhOiAoTY6LQNzG+elvPdWwbkcWfasJPK2Yf8XON+Dm2j8jOa8RzZiEx/nJP+CW2kcjYRnjxs1G2Ec+//k/8/dxOYnx9cdeJNxcbG4lXd9348WpYRD/koTI6F+NvP4nfYjYoYpBmRWJGvPx4eyZe3jVqIEQhHK45L0UrkcV/1DpxMESeTZSs/loaEG/fGk+HRsxzLnyjA5Wxy093/z0dFNH8rZ+x6fX6d4xNoywkxlU3fl1tGmUhka3+fHu9YVlJjK8uLuNNo2wkstXp6l6jrCTWGwXigIg3Nxkb0u1NxnGfxCAiIqlClJxRFYdcxzsiMh5RlaPODtRpc1hEFKZuMqFKwqUq3gmR95MDotLdgbrLuUis80rPxT243IAIIogggggiiCCCCCKIIIIIIogggghij8TRrHkii1lETcsxk3Nq9guH2Nh4RhJ/e3aik1CjuR1ynPcO6yTehMqIrnTUixG1gzOqMKfb68ypindI5HhCVQ5be1FF6TIoi6RsOVDJc5ERFfJu+Ppjm5c9FzNf8FxyYfHlJis115I7a6GdRHUmdORraTruZoBEvC+CCCKIIIIIIogggggiiCCCCOKeE7sdj/rCd+AMe5ERY4fpUcVVLzYPRJ46Y2Id5p30RnyhiE5nvO7sMw3qg4uZnFMfXThHr/AzjX243IAIIogggggiiCCCCCKI3RFVrsJcqdxiIs/9RPFAaYuJmZ9Jf/0kLWvH3fAiVH6epBb3oiqyrKi+2nwu2n9FBRFEm4gd3kedvc77qI7YPvQ6dIWebbuzecPpXiTnu2Hbz3fjvNpJtbc+FzFTCogggggiiCCCCCKIIIIIIogggggiiCAaEkfUNNi73GTc5cl9P56dSM6xEVCFyRNzbJDreO+J/PX3r1S+d0OkZz0hC0/NoUJWvHlzfvzzhsy7rg7Unue7+fD8xL4vNyDuPTGXWSBVZPYorWESeRaoPFdarb+zk1jqrNRFqTn3DZ4WNkhioENHc5nbfKDaf7kBEUQQQQQRRBBBBBFEEEHsi/jEXMU0scOZUl6iF4mZUp54/qIkn7/obP38xX8/vPmFSFfE3seGf39HhhJiMhgQQQQRRBBBBBFEEEEEEUQQQbSfWOhSF1wvLSby9YCNMNOKq8JWokhDWfiLkpfaUqLMSz9QztMDxIZNLP3UV7xUFhNrV9Qdbpd2mi6JXiNx2uEebnez5XJHbZm1LrFZ7qSByPXDHxAplOGCphs0XY4vttzgI2JhuIHMlGi6QdPleLblBh8RDWMKNF7WbIPq7s8Wm3tI9IUw2VWw9J2gfbGFTBOppNO2XJ74MjRYbj3+ORF+0npQZws9yrjjpPwxUaukdT9VnDw3Onn0UuWLm+HKLctVxKLM0/YNSq5z3t4NSgSLVJfrg/UhMVeydfV1wrnRa6GzVCz99hZVL1eU+e0vRbasfIu0fYPBTU/6GX9MVGLZunqVhUiFwZVtGYTVYknrlaQIlnrBnfYrzpkuk6VKstYNJpnWvBDr12zXy82AAqIN2QPif77bdKDAKgh1AAAAAElFTkSuQmCC\" width=\"400\" />\n",
+    "</p>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Mq5SNtn4Alur"
+   },
+   "source": [
+    "Here, we jusy install RLHive. "
```

Comment:
*just
It might be a good idea to double-check the text part with Grammarly.

Reply:
done

Reply:
checked also with Grammarly.